saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SemanticRoleLabeling/PopulateSRLDataModel.scala

```diff
@@ -111,7 +110,6 @@ object PopulateSRLDataModel extends Logging {
         logger.debug("gold relations for this train:" + gr.relations().size)
         if (!useGoldArgBoundaries) {
           val XuPalmerCandidateArgsTraining = gr.predicates.getTrainingInstances.flatMap(x => xuPalmerCandidate(x, (gr.sentences(x.getTextAnnotation) ~> gr.sentencesToStringTree).head))
-          gr.sentencesToRelations.addSensor(textAnnotationToRelationMatch _)
```

Comment:
We already have this matching sensor in the DataModel. https://github.com/IllinoisCogComp/saul/blob/master/saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SemanticRoleLabeling/SRLMultiGraphDataModel.scala#L51